### ISSUE TITLE
Cannot get output of Build/CreatePackage targets for *.nuproj

### ISSUE THREAD

**Issue by AndirNotes** [terrajobst/nuproj#82]
I am trying to use NuProj with MSBuild build scripts and I can't get any output from targets Build and/or CreatePackage. My intention is to get all *.nupkg from output and copy them to special place.

I have something like: 
```
<Project ToolsVersion="4.0"
         DefaultTargets="Build"
         xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
    <ItemGroup>
        <_ProjectsToBuild Include="My.nuproj" />
    </ItemGroup>

    <PropertyGroup>
        <CustomAfterMicrosoftCommonTargets>$(MSBuildExtensionsPath)\12.0\Microsoft.Common.Targets\ImportAfter\Microsoft.Common.NuProj.targets</CustomAfterMicrosoftCommonTargets>

        <_ProjectsToBuildProperties>
            CustomAfterMicrosoftCommonTargets=$(CustomAfterMicrosoftCommonTargets);
        </_ProjectsToBuildProperties>
    </PropertyGroup>

    <Target Name="Build">
        <MSBuild Projects="@(_ProjectsToBuild)"
                 Properties="$(_ProjectsToBuildProperties)">
            <Output TaskParameter="TargetOutputs"
                    ItemName="_ProjectsToBuildOutput" />
        </MSBuild>

        <Message Text="BuildOutput: @(_ProjectsToBuildOutput)." />
    </Target>

</Project>
```

**Comment by williamb1024** [comment]
I had a similar issue and have made a pull request that adds returns for the Build target. It's pretty simple if you want to add directly to your installed copy.

**Comment by AndirNotes** [comment]
Ok, that sounds good. 
But I'm slightly confused about CreatePackage target. It has Outputs parameter, but anyway I have nothing in the result if I call it. What's special about the Returns parameter?

**Comment by williamb1024** [comment]
The Build target in nuproj.targets does not have Outputs or Returns attribute, so it does not return anything in TargetOutputs. The Returns attribute will specify what should be returned in the TargetOutputs.

**Comment by AndirNotes** [comment]
I understood your solution, I was also trying to use CreatePackage target from *.nuproj file. This target which could be located in the NuProj.targets file has "Outputs" attribute, so it should also return results in the TargetOutputs, but it doesn't work in that way by some reasons.

**Comment by williamb1024** [comment]
Yes, I agree it should. It wouldn't work correctly as it exists anyway. CreatePackage may create more than one nupkg file, depending on the setting for generating symbol packages, but the outputs would only contain the primary nupkg file.

**Comment by AndirNotes** [comment]
Ok, I checked you pull-request (#70) - I agree - it is better solution. But I think you need to extend it to the CreatePackage target too.
Also I figured out why CreatePackage Outputs is not working - it seems that this is a common behavior of MSBuild, if script contains any target with Returns attribute (even not used at all) - any other target Outputs will be ignored (only Returns will be used). I can't find any documentation on that, so I asked about the issue on stackoverflow, maybe someone knows http://stackoverflow.com/questions/28851164/msbuild-confusing-behavior-with-target-outputs-returns .
